sdk/attestation/azure-security-attestation/azure/security/attestation/aio/_client_async.py

```diff
@@ -0,0 +1,90 @@
+# coding=utf-8
```

Comment:
I thought the # coding=utf-8 was a python-ism indicating that the source should be rendered as UTF-8, is that not the case?

Reply:
The encoding is fine, but this file looks like generated code, are you using this for your convenience layer?

Reply:
Yes - because the convenience layer started with the generated code (because I trust the generated code to create all the python idiosyncrasies properly, and I have no faith in my  ability to do the same).